protocols/identify/src/handler.rs

```diff
@@ -120,8 +149,9 @@ impl ConnectionHandler for IdentifyHandler {
             EitherOutput::Second(fut) => {
                 if self.inbound_identify_push.replace(fut).is_some() {
                     warn!(
```

Comment:
Unrelated: We might want to upgrade to `tracing` at some point so we can emit these values in a structured way.

Or we wait until `log` has a structured key-value support.

Reply:
:+1: for structured logging, though not something I will focus on any time soon. Don't have an opinion on crates providing structured logging.